docs/troubleshooting/index.mdx

```diff
@@ -35,6 +37,72 @@ Alpine is missing some dependencies that [are required](https://github.com/datre
 Adding the following command to the CI should fix this: `apk add curl bash`
 This will add curl and bash commands which our install scripts require in order to run.
 
+---
+
+## Schema validation failure
+
+**TL;DR**  
+If you get the following message in your CLI: 
+```terminal
+❌  k8s schema validation error: could not find schema for [kindName]  
+``` 
+It means the schema validation check failed and there are three possible root causes:
+* [Worng Kubernetes schema version](troubleshooting#worng-kubernetes-schema-version) - you need to update the schema version on your dashboard
+* [Deprecated API versions](troubleshooting#deprecated-api-versions) - you need to fix your manifest
+* [Missing schema for a CRD](troubleshooting#missing-schema-for-a-crd) - you need to use the CRD Extractor
```

Comment:
perhaps turn "CRD extractor" into a link

Reply:
It's the tl;dr, so I don't think it will make sense to give the link w/o (enough) context